docs/CHARTER.md

```diff
@@ -0,0 +1,114 @@
+# Technical Charter (the “Charter”) for Package Analysis / Feeds a Series of LF Projects, LLC
+
+**Adopted ___________**
+
+This Charter sets forth the responsibilities and procedures for technical contribution to, and oversight of, the Package Analysis / Feeds open source project, which has been established as Package Analysis / Feeds a Series of LF Projects, LLC (the “Project”).  LF Projects, LLC (“LF Projects”) is a Delaware series limited liability company. All contributors (including committers, maintainers, and other technical positions) and other participants in the Project (collectively, “Collaborators”) must comply with the terms of this Charter. 
+
+
+
+## 1. Mission and Scope of the Project
+
+a. The mission of the Project is to improve the security of open source software by detecting malicious behavior, informing consumers selecting packages, and providing researchers with data about the ecosystem.
```

Comment:
a. The mission of the Project is to improve the security of open source software by detecting malicious and unsafe behaviour in packages published to open source repositories, helping inform consumers about the packages they choose, and providing security teams and researchers with data about open source ecosystems.